server/server.go

```diff
@@ -192,7 +197,29 @@ func NewEmulatorServer(logger *logrus.Logger, conf *Config) *EmulatorServer {
 	return server
 }
 
+// Listen starts listening for incoming connections.
+//
+// After this non-blocking function executes we can treat the
+// emulator server as ready.
+func (s *EmulatorServer) Listen() error {
+	s.listeners = make([]listener, 0)
+	s.listeners = append(s.listeners, s.grpc)
+	s.listeners = append(s.listeners, s.rest)
+	s.listeners = append(s.listeners, s.admin)
```

Comment:
Maybe shorten this a bit:

```suggestion
	s.listeners = []listener{s.grpc, s.rest, s.admin}
```

Reply:
lol